modules/hal_hpmicro/Kconfig

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 hpmicro
+# SPDX-License-Identifier: Apache-2.0
+config ZEPHYR_HAL_HPMICRO_MODULE
+	bool
+config HAS_HPMICRO_HAL
+	bool "HPMSDK HAL drivers support"
+config HAS_HPMSDK_UART
+	bool "HPMSDK UART driver support"
+config HAS_HPMSDK_DRAM
+	bool "HPMSDK DRAM driver support"
+config HAS_HPMSDK_SDP
+	bool "HPMSDK SDP driver support"
```

Comment:
options to enable certain HAL features are tipically named "USE_", to indicate e.g. that a driver uses certain HAL modules. "HAS_" is tipically used to indicate a certain SoC has a feature. You can check GD32/STM32 for reference.

Reply:
Yes, these configurations are passed to hpm_sdk for the purpose of trimming the functionality of the soc, so using `HAS` here I think would make more sense, again https://github.com/zephyrproject-rtos/zephyr/pull/46329#discussion_ r895501958 's comment is the same as this one, HAS_X is placed at the soc to indicate whether the SOC supports this feature.